RELEASE-LEADS.md

```diff
@@ -75,29 +75,19 @@ https://github.com/knative/community/blob/master/peribolos/knative-sandbox.yaml
 [this one](https://github.com/knative/community/pull/209) to grant yourself some
 super power.
 
-### Revert all pins to pin master branches again
```

Comment:
Unless I missed it in the diff, there's now no un-pin step to get us back to tracking master branches?

Reply:
We no longer pin and unpin things. That is handled dynamically based on the release you pass to the update-deps script.

Reply:
Im not quite following.. at some point doesn't the automation have to go back to bumping e.g. serving to the master version of pkg rather than the release branch?

Reply:
Ok yes, there is a missing step to bump what knobots is using... Let me add that.

Reply:
Well, good catch, https://github.com/knative/hack/pull/8